tests/foreman/ui/test_bookmark.py

```diff
@@ -409,15 +392,13 @@ def test_negative_update_bookmark_name(self):
                             query=gen_string(random.choice(STRING_TYPES)),
                         )
                     self.bookmark.update(
-                        entity['controller'],
                         bm2_name,
                         bm1_name,
                         gen_string(random.choice(STRING_TYPES)),
                     )
                     self.assertTrue(self.bookmark.wait_until_element(
                         common_locators['name_haserror']))
```

Comment:
Yes, as this check is performed from Bookmarks page and `edit bookmark` window / validation is the same for any controller as opposed to the create window.
Anyway, test results with `for entity in self.entities:`:
```
2017-06-08 14:44:56 - robottelo - DEBUG - Finished Test: BookmarkTestCase/test_negative_update_bookmark_name
.
2017-06-08 14:54:03 - robottelo - DEBUG - Finished Test: BookmarkTestCase/test_negative_update_bookmark_name_empty
.
================== 2 passed, 12 deselected in 1236.32 seconds ==================
```